configurator/Dockerfile

```diff
@@ -1,18 +1,21 @@
 FROM node:16-alpine
 
-RUN apk add g++ make py3-pip git curl chromium
-
 WORKDIR /scripts/configurator
 
-COPY ./configurator ./
+COPY ./configurator/package.json ./package.json
+COPY ./configurator/package-lock.json ./package-lock.json
 
 RUN npm install
 
 WORKDIR /scripts/cht-config
 
 COPY ../cht-config ./
 
-RUN npm install && npm install -g cht-conf && python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
```

Comment:
`python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic` is still needed. I'm getting `ERROR Error: There was a problem executing xls2xform.  It may not be installed.` in the configurator.

Reply:
trying to get the Dockerfile to run quicker, but not possible I guess. added it back